lib/sqs.js

```diff
@@ -4,6 +4,14 @@ const _ = require('lodash');
 const AWS = require('aws-sdk');
 const Promise = require('bluebird');
 const clientConfig = require('./client-config');
+const uuid = require('uuid/v4');
+const util = require('util');
+const zlib = require('zlib');
+const compress = util.promisify(zlib.gzip);
+const decompress = util.promisify(zlib.gunzip);
+
+// 256kb => b
+const TWO_FIFTYSIX_K = 256 * 1024;
```

Comment:
heh maybe being kind of annoying here but can we give this constant a meaningful name?  MAX_MSG_SIZE or something?